modules/distribution/src/repository/resources/dbscripts/apimgt/oracle_rac.sql

```diff
@@ -243,7 +245,7 @@ CREATE TABLE SP_APP (
             USER_STORE VARCHAR (255) NOT NULL,
             USERNAME VARCHAR (255) NOT NULL ,
             DESCRIPTION VARCHAR (1024),
-                ROLE_CLAIM VARCHAR (512),
+	    	    ROLE_CLAIM VARCHAR (512),
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04